testdata/project-v3-with-kustomize-v2/config/crd/bases/crew.testproject.org_admirales.yaml

```diff
@@ -5,20 +5,20 @@ metadata:
   annotations:
     controller-gen.kubebuilder.io/version: v0.9.0
   creationTimestamp: null
-  name: admirals.crew.testproject.org
+  name: admirales.crew.testproject.org
```

Comment:
Doing this in sample is fine. But still I have to say the plural change may lead to a high risk in the real world.

For example, the old CRD `admirals.crew.testproject.org` was already applied into K8s cluster. Then we generate a new `admirales.crew.testproject.org` CRD, how should we update it?
1. If we just apply it into cluster, there will be two CRDs with different name and plural, but the same group, version and kind.
2. If we remove the old CRD and then apply the new one, all existing CRs of this CRD will be removed.

So, I'm just afraid will this be a bad sample for users to manage their CRDs? Or we have to warn here that ppl should not change the plural unless they know what they are doing.

Reply:
The purpose of the PR is only to ensure that in both tests we are doing exactly the same check

Why do we scaffold it in the testdata sample?

Because we need to ensure that changes in the code will not break the feature. Only because of that. 
The samples under testdata are used only to check the layouts and they have no implementation or are used as examples.

Reply:
Emm.. I'm ok with this since they are used only to check the layouts and have no implementation or are used as examples.

LGTM, let's see if @erikgb has any ideas. (and it has conflicts should be resolved)

Reply:
I'm ok with this for now, after the clarifications from @camilamacedo86.

Reply:
Thank you folks.